packages/lib/src/user/utils.ts

```diff
@@ -252,30 +258,32 @@ const hasAudience = (
   // The user affiliation was found but not the campus, don't show this
   if (!hasValue(locations, campusName)) return false;
 
+  // The user affiliation and campus were found, audiences empty is treated as reason to show all and should
+  // skip any further evaluation.
+  if (audiences?.length === 0) return true;
+
+  // Employee in the employee dashboard
+  if (isEmployeeOnly(user) && usersAffiliation === AFFILIATIONS.employee) {
+    return affiliation.some(a => AFFILIATIONS.employee.toLowerCase() === a.toLowerCase());
+  }
+
   const usersAudiences: string[] = [];
 
   // This is an employee (no user.classification.attributes), using the Student Dashboard (usersAffiliation is finding
-  // the affiliationOverride set), and they have no audienceOverrides set in thier profile, then default to returning
-  // true if the item is an undergrad audience.. Undergrad is considered the default audience.
-  if (isEmployeeOnly(user) && usersAffiliation === AFFILIATIONS.student) {
+  // the affiliationOverride set). Ensure that a default audience of undergraduate exists unless they have an
+  // audience override specifying to see graduate student items.
+  if (
```

Comment:
Refactored this to combine the if logic into an if-statement that covers the scenarios that we want to pre-fill the undergraduate audience.. no need to return as the following methods now take into consideration if the user `isEmployeeOnly`